src/App.tsx

```diff
@@ -1,16 +1,24 @@
-import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
+import './App.scss';
 
 export const App = () => {
+  const [newMovieList, setNewMovieList] = useState(moviesFromServer);
+
+  const onAdd = (newMovie: Movie) => {
```

Comment:
```suggestion
  const handleMovieAdd = (newMovie: Movie) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364